web/src/components/identity/Access.tsx

```diff
@@ -44,14 +43,13 @@ class Access extends Component {
               <Route component={NotFound} />
             </Switch>
           </Fragment> */}
-          <IdentityProviders
-            isKurlEnabled={this.props.isKurlEnabled}
-            isGeoaxisSupported={this.props.isGeoaxisSupported}
-          />
-        </div>
+        <IdentityProviders
+          isKurlEnabled={props.isKurlEnabled}
+          isGeoaxisSupported={props.isGeoaxisSupported}
+        />
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
 
-export default withTheme(withRouter(Access));
```

Comment:
I don't see any props from `withTheme` being used here or passed down to other components. LMK if removing this is an error